lib/fiwareDeviceSimulator.js

```diff
@@ -47,6 +47,24 @@ var fiwareDeviceSimulatorValidator = require(ROOT_PATH + '/lib/validators/fiware
 var fiwareDeviceSimulatorCollector = require(ROOT_PATH + '/lib/collectors/fiwareDeviceSimulatorCollector');
 
 /**
+ * Week day names in distict languages.
+ */
+var DAYS = {
+  en: ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday'],
+  es: ['domingo', 'lunes', 'martes', 'miércoles', 'jueves', 'viernes', 'sábado']
+};
+
+/**
+ * Months names in distict languages.
+ */
+var MONTHS = {
+  en: ['january', 'february', 'march', 'april', 'may', 'june', 'july', 'august', 'september', 'october',
+    'november', 'december'],
```

Comment:
The "months template" sets the final value all to uppercase or all to lowercase so it is not really that important here. Would you like me to include them with a starting uppercase letter anyhow??? :)